qlib/workflow/recorder.py

```diff
@@ -223,6 +220,15 @@ def __init__(self, experiment_id, uri, name=None, mlflow_run=None):
                 else None
             )
 
+    @property
+    def fm(self):

```

Comment:
I just happen to see this line . Why is `self.fm.save_obj(data, name)` necessary here?  
![image](https://user-images.githubusercontent.com/465606/102983397-fc6d9780-4546-11eb-8b54-b7d4debf2b10.png)

Reply:
yes, I have just removed it to make it simple.